decoder.rb

```diff
@@ -0,0 +1,35 @@
+# decode morse code character
+
+def decode_char(character)
+  # dictionary
+  dictionary = {
+    'A' => '.-', 'B' => '-...', 'C' => '-.-.', 'D' => '-..', 'E' => '.', 'F' => '..-.',
+    'G' => '--.', 'H' => '....', 'I' => '..', 'J' => '.---', 'K' => '-.-', 'L' => '.-..',
+    'M' => '--', 'N' => '-.', 'O' => '---', 'P' => '.--.', 'Q' => '--.-', 'R' => '.-.', 'S' => '...',
+    'T' => '-', 'U' => '..-', 'V' => '...-', 'W' => '.--', 'X' => '-..-', 'Y' => '-.--', 'Z' => '--..'
+  }
+  dictionary.each do |key, value|
+    return key if character == value
+  end
+end
+
+def decode_word(word)
+  split_word = word.split
+  split_word.each do |i|
+    decode_char(i)
+  end
+end
```

Comment:
Hi There 👋🏾 ,
Currently, your  decode word function does not return the correct option and returns back the morse code inputted
![image](https://user-images.githubusercontent.com/60612329/208966771-eff88a31-9169-49eb-a227-a71450f6fe15.png)

Reply:
Hello Alexander,
Thank you for the constructive feedback on our team project.
Issues resolve now.